src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/multinomial.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_op_tests/multinomial.hpp"
+
+#include <openvino/core/type/element_type.hpp>
+#include <openvino/runtime/tensor.hpp>
+
+namespace {
+
+using ov::test::MultinomialLayerTest;
+
+const int GLOBAL_SEED = 1;
+const int OP_SEED = 2;
+
+std::vector<float> probs_4x4_f32 = {0.00001f,
```

Comment:
Do these number have a certain meaning? Can we rely here on the random generator as usual?